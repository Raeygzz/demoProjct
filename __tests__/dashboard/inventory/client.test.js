import 'react-native';
import React from 'react';
import Client from '../../../src/screens/dashboard/inventory/client';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Client />).toJSON();
  expect(tree).toMatchSnapshot();
});
