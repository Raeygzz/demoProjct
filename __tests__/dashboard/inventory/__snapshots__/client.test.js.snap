// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "flex-end",
            "marginBottom": 20,
          }
        }
      >
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          onSubmitEditing={[Function]}
          placeholder="Search Client"
          rejectResponderTermination={true}
          style={
            Array [
              Object {
                "backgroundColor": "white",
                "height": 40,
                "left": 0,
                "paddingLeft": 20,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "zIndex": 2,
              },
              Object {
                "fontFamily": "Apercu-Medium",
              },
            ]
          }
          underlineColorAndroid="transparent"
        />
        <View
          style={
            Object {
              "marginRight": 12,
              "marginTop": 10,
              "zIndex": 999,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 20,
                },
                Object {
                  "color": "black",
                },
                Object {
                  "fontFamily": "simple-line-icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            ÓÇê
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginVertical": 15,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
